ports/esp32/boards/SPARKFUN_IOT_REDBOARD_ESP32/pins.csv

```diff
@@ -0,0 +1,45 @@
+GP1,GPIO1
+GP2,GPIO2
+GP3,GPIO3
+GP4,GPIO4
+GP5,GPIO5
+GP13,GPIO13
+GP14,GPIO14
+GP16,GPIO16
+GP17,GPIO17
+GP18,GPIO18
+GP19,GPIO19
+GP21,GPIO21
+GP22,GPIO22
+GP23,GPIO23
+GP25,GPIO25
+GP26,GPIO26
+GP27,GPIO27
+GP32,GPIO32
+GP33,GPIO33
+GP34,GPIO34
+GP35,GPIO35
+GP36,GPIO36
+GP39,GPIO39
```

Comment:
From what I can see on the pics of this board, there are no pins labelled "GP1" etc.  Instead they are just labelled by their number, eg "1".  So I don't think any of these defines above are needed (because specifying the pin number like `machine.Pin(1)` already works on all esp32 boards).